modules/ROOT/pages/migration-munit-test-structure.adoc

```diff
@@ -0,0 +1,46 @@
+= Migrating Test Structure (MUnit)
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The MUnit test is divided in three scopes. All of them are optional:
+
+[%header,cols="20,80"]
+|===
+|Scope |Description
+
+| Behavior
+| The behavior scope is meant to set all the preconditions before executing the test logic. +
```

Comment:
Is the behavior scope an XML element (<munit:behavior/>)? If so, we can use code format for it. Do we need an XML example for this?

Reply:
Yes, each new scope is an XML element that contains other processors inside of it. I will change the example to have all these 3 sections.